.github/workflows/test-short.yml

```diff
@@ -0,0 +1,20 @@
+name: test-short
```

Comment:
[blocking] Proposal to add this job to the `test.yml` file that already exists at https://github.com/celestiaorg/celestia-app/blob/c90e61d5a2d0c0bd0e123df4ab416f6f0d141b7f/.github/workflows/test.yml

Reply:
Updated! Thanks